service.go

```diff
@@ -171,11 +179,13 @@ func initializeHealthCheckService() *k8sHealthcheckService {
 	}
 
 	services := make(map[string]service)
+	sysCodeFetchGuard := make(chan struct{}, 15)
```

Comment:
**question:** Is the idea behind `sysCodeFetchGuard` to be able to fetch the systemCode of at most 15 services concurrently ?